api/pyproject.toml

```diff
@@ -34,4 +34,4 @@ known_third_party=['_pytest','apiclient','app_analytics','axes','chargebee','cor
 skip = ['migrations','.venv','.direnv']
 
 [tool.pytest.ini_options]
-addopts = ['--ds=app.settings.test', '-vvvv', '-p', 'no:warnings', '-n', 'auto']
```

Comment:
I guess I'm still owing to fix this, but it doesn't belong in this PR.

Reply:
Sorry! Yes.